samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -80,6 +86,15 @@ public long getOffsetFileTTL() {
    *         on the job status server.
    */
   public List<String> getJobStatusServers() {
-     return Arrays.asList(StringUtils.split(get(CONFIG_JOB_STATUS_SERVERS), '.'));
+     return Arrays.asList(StringUtils.split(get(CONFIG_JOB_STATUS_SERVERS), ','));
+  }
+
+  /**
+   *
```

Comment:
Added. 

> please add description for javadocs.

Just a curious question. Wondering does it makes sense to have java doc for these config methods. They're just wrapper on top of get methods and has default value if undefined(doesn't have complicated logic that need explanation, code is plain obvious) and the name is self explanatory. Just thinking, what's the value addition by these docs/explanation? Please share your thoughts.